### PR TITLE
marker appareance

### PR DESCRIPTION
**Pull request by pgilliar**
added a default icon to marker and a background

### CODE REVIEW

**Comment by zaclegarssure** [comment]
While it looks cool, is it really impossible to test this ?

File: app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsFragment.kt [hunk 1]
```diff
@@ -282,10 +289,20 @@ class MapsFragment : Fragment(), OnMapReadyCallback, GoogleMap.OnMarkerClickList
             val position = LatLng(meetUp.location.latitude, meetUp.location.longitude)
             val options = MarkerOptions().position(position).title(meetUp.uuid)
 
-            if (meetUp.markerId != null && iconPack != null) {
-                val icon = iconPack!!.getIcon(meetUp.markerId)
-                val bitmap = icon?.drawable?.toBitmap(64, 64)
-                if (bitmap != null) options.icon(BitmapDescriptorFactory.fromBitmap(bitmap))
+            val markerId = meetUp.markerId ?: 2
```
Comment (zaclegarssure) [inline app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsFragment.kt:292]: Also here the 2, maybe add a constant named DEFAULT_MARKER_ID

**Review by Gerussi — APPROVED** [review]
Very well commented, good job!

File: app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsFragment.kt (hunk 1 above)
Comment (pgilliar) [reply app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsFragment.kt]: It's fixed and I wille test it hext sprint

**Review by zaclegarssure — APPROVED** [review]
Otherwise it's good.

**Pull request merged**